src/power_grid_model/utils.py

```diff
@@ -54,6 +55,35 @@ def _get_component_scenario(component_scenarios: BatchArray) -> np.ndarray:
     return {component: _get_component_scenario(component_data) for component, component_data in dataset.items()}
 
 
+def get_data_set_batch_size(dataset: BatchDataset) -> int:
+    """
+    Get the number of scenarios in the batch dataset.
+
+    Args:
+        dataset: the batch dataset
+
+    Raises:
+        ValueError: if the batch dataset is inconsistent.
+
+    Returns:
+        The size of the batch dataset. Making use of existing _utils function.
+    """
+    return get_and_verify_batch_sizes(dataset)
```

Comment:
why not do `... import get_and_verify_batch_sizes as get_dataset_batch_size`? is the docstring crappy?

Reply:
Because 
`pylint:
src\power_grid_model\utils.py:17:0: W0611: Unused get_and_verify_batch_sizes imported from power_grid_model._utils as get_data_set_batch_size (unused-import)
`
after ` # pylint: disable=unused-import`

Reply:
I'll stick to the way that's in the last commit. The following code piece causes issues between isort and pylint.
Original:
`
from power_grid_model._utils import get_and_verify_batch_sizes as get_data_set_batch_size,  # pylint: disable=unused-import
from power_grid_model._utils import get_batch_size as get_data_array_batch_size  # pylint: disable=unused-import
`
isort would fix this to:
`
from power_grid_model._utils import (
    get_and_verify_batch_sizes as get_data_set_batch_size,  # pylint: disable=unused-import
)
from power_grid_model._utils import get_batch_size as get_data_array_batch_size  # pylint: disable=unused-import
`
and then pylint would complain about the unused first import.
On the other hand, user would easily go to _utils.py when checking for definition of `get_data_set_batch_size()` and `get_data_array_batch_size ()`